src/renderer/Dataframe.js

```diff
@@ -164,106 +268,129 @@ export default class Dataframe {
         return { minx, maxx, miny, maxy };
     }
 
-    _getPointsAtPosition (p, viz) {
-        p = wToR(p.x, p.y, { center: this.center, scale: this.scale });
+    _getViewportPoints (viewportAABB) {
+        return [
```

Comment:
This seems like a smell to me, is there a reason to have this method?

Reply:
I moved this logic to `collision`. The only reason to do that was that for some operations (`pointInTriangle` and `signedDistanceFromPointToLine`) we need the viewport coordinates instead of the AABB.